nucypher/cli/painting.py

```diff
@@ -715,3 +715,109 @@ def write_deployed_allocations_to_csv(filepath: str, allocated: list, failed: li
             name = allocation.get('name', 'No name provided')
             row = (beneficiary, name, contract_address)
             writer.writerow(dict(zip(fieldnames, row)))
+
+
+def paint_worklock_status(emitter, registry: BaseContractRegistry):
+    from maya import MayaDT
+    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)
+    blockchain = worklock_agent.blockchain
+
+    # Agency
+    token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=registry)
+
+    # Time
+    start = MayaDT(worklock_agent.contract.functions.startBidDate().call())
+    end = MayaDT(worklock_agent.contract.functions.endBidDate().call())
+    duration = end - start
+    remaining = end - maya.now()
+
+    # TODO: Include calculated refund and deposit rates
+
+    payload = f"""
+
+Time
+======================================================
+Start Date ........ {start}
+End Date .......... {end}
+Duration .......... {duration}
+Time Remaining .... {remaining} 
+
+Economics
+======================================================            
+ETH Pool .......... {blockchain.client.get_balance(worklock_agent.contract_address)}
```

Comment:
Not sure about this value, because it's not ok to use it as indicator for rates calculation but useful to show how many ETH still locked and not yet refund (only after end of bidding)

Reply:
I am open to naming suggestions here.

Reply:
How about also add something like `ETH supply ...... worklock_agent.get_eth_supply` and leave `ETH Pool` unchanged